composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "cboden/ratchet"
+    "name": "minifets/ratchet"
```

Comment:
I think you forget to remove this diff :)

Reply:
Sorry, I added the changes after the submiting pull request.